src/_locales/en/messages.json

```
@@ -0,0 +1,109 @@
+{
+  "launchInExternalBrowser": {
+    "message": "Open page in $browser$",
+    "placeholders": { "browser": { "content": "$1" } }
+  },
+  "launchInExternalBrowserLink": {
+    "message": "Open link in $browser$",
+    "placeholders": { "browser": { "content": "$1" } }
+  },
+  "welcomeHeading": {
+    "message": "Welcome to Firefox Launch"
+  },
+  "manageExternalSitesContextMenus": {
+    "message": "Manage My External Sites"
+  },
+  "autoRedirectCheckboxContextMenu": {
+    "message": "Auto-redirect My External Sites"
+  },
+  "addCurrentSiteToMySitesContextMenu": {
+    "message": "Add $sld$ to My External Sites",
+    "placeholders": { "sld": { "content": "$1" } }
+  },
+  "changeDefaultLaunchContextMenu": {
+    "message": "Always use Firefox Private Browsing"
+  },
+  "notInstalledErrorHeading": {
+    "message": "Firefox is not installed"
+  },
+  "notInstalledErrorText": {
+    "message": "Firefox Launch requires Firefox to be installed on your computer."
+  },
+  "notInstalledErrorAction": {
+    "message": "Download Firefox"
+  },
+  "openWelcomePage": {
+    "message": "Extension Settings"
+  },
+  "welcomePageTitleFirefox": {
+    "message": "Launch in Another Browser"
+  },
+  "welcomePageSubtitleFirefox": {
+    "message": "Switch between browsers in one click."
+  },
+  "welcomePageDescriptionFirefox": {
+    "message": "Sometimes, you just want to use a different browser.\nThis extension helps you easily open sites in other browsers."
+  },
+  "welcomePageTitleChromium": {
+    "message": "Launch in Firefox"
+  },
+  "welcomePageSubtitleChromium": {
+    "message": "Switch from your current browser to Firefox in one click."
+  },
+  "welcomePageDescriptionChromium": {
+    "message": "Sometimes, you just want a more private browser. This extension helps you easily open sites in Firefox."
+  },
+  "welcomePageBrowserSelector": {
+    "message": "Select other browser"
+  },
+  "welcomePageAlwaysPrivateCheckbox": {
+    "message": "Always open sites in a Firefox Private window when switching"
+  },
+  "welcomePageTelemetryCheckbox": {
+    "message": "Allow Mozilla to collect information about how you use this extension so we can make it better. {LinkStart}Privacy Notice{LinkEnd}"
+  },
+  "welcomePageNoShortcutsFirefox": {
+    "message": "You don't have any shortcuts set up for this extension. Create them by going to {LinkStart}extension settings.{LinkEnd}"
```

```suggestion
    "message": "You don’t have any shortcuts set up for this extension. Create them by going to {LinkStart}extension settings.{LinkEnd}"
```

This and the next two strings need to use the correct apostrophe.

---

PR: https://github.com/mozilla/firefox-launch/pull/21